manifests/service.pp

```diff
@@ -16,12 +17,14 @@
     enable => $jobs_service_enable,
   }
 
-  if $passenger {
-    exec {'restart_foreman':
-      command     => "/bin/touch ${app_root}/tmp/restart.txt",
-      refreshonly => true,
-      cwd         => $app_root,
-      path        => '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
+  if $apache {
+    if $passenger {
+      exec {'restart_foreman':
```

Comment:
Where is this used?

Reply:
It's used implicitly. It's a `refreshonly` exec, but when `Class['foreman::service']` is refreshed, this `exec` is refreshed. Note it's been there for ages.